scripts/deployment/slaves.in

```diff
@@ -1,3 +0,0 @@
-18.237.237.45
```

Comment:
we could consider remove this file. users could add it themselves

Reply:
Removed